data/config_file_generators/realization_config_generator.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+"""
+Script to generate a realization configuration JSON file with randomly selected
+formulations for each catchment given a catchment data geojson file as input. 
+
+Script and command line arg format:
+realization_config_generator.py -i <path and file name to catchment data geojson file>
+-o <file name of config output file>
+
+Example:
+python realization_config_generator.py -i ../catchment_data.geojson -o realization_config.json
+
+TODO: Determine method to pass each catchment's forcing file to catchment JSON object
```

Comment:
@donaldwj @hellkite500 I am unclear about what the current consensus is for feeding the forcing for each catchment for the demo. Is it through ngen or BMI? If it is through ngen, then I can find a method to pass each catchment's forcing file to this config.

Reply:
The discussion this morning clarified things for me more on this. The existing structure should allow to set forcing at the top, and each catchment would use it.